### PR TITLE
Validate resource and CoffeePicker api

### PR DESCRIPTION
**Pull request by rdjpalmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,8 +51,8 @@ pact.tokens
   - `create()`
   - `del()`
 - products
-  - `list()`
```
Comment (iest) [inline README.md]: We still have list right?

File: README.md [hunk 2]
```diff
@@ -78,6 +78,10 @@ pact.tokens
 - identities
   - `create({params})`
   - `update({params})`
+- validate
+  - `password({params})`
```
Comment (iest) [inline README.md]: Is it `{params}`? Or `password`

Same with voucher.

**Comment by iest** [comment]
Code's fine.
:ship: once the readme's updated.

File: README.md (hunk 2 above)
Comment (rdjpalmer) [inline README.md]: It's hash that gets passed in as the payload to the `POST` request. `{password}` may be more accurate.

File: README.md (hunk 1 above)
Comment (rdjpalmer) [inline README.md]: Yep, my mistake.

**Pull request merged**